app/pages/project/metadata.cjsx

```diff
@@ -1,12 +1,31 @@
 React = require 'react'
 {Link} = require 'react-router'
+visible = (require 'visible-element')()
 
 module?.exports = React.createClass
   displayName: 'ProjectMetadata'
 
   propTypes:
     project: React.PropTypes.object
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  scrollDone: false
+
+  handleScroll: ->
+    return if @scrollDone
+
+    @scrollDone = visible.inViewport(@refs.belowStats)
+    @context.geordi?.logEvent type: 'scroll-down' if @scrollDone
+
+  componentDidMount: ->
```

Comment:
@eatyourgreens I assume using debounce would accomplish this? Instead of running `handleScroll` with every tiny scroll, we could control how many milliseconds pass until `handleScroll` is called.